src/toil_scripts/tools/__init__.py

```diff
@@ -0,0 +1,26 @@
+import os
+import subprocess
+
+
+def get_mean_insert_size(work_dir, bam_name):
+    """Function taken from MC3 Pipeline"""
+    cmd = "docker run --log-driver=none --rm -v {}:/data quay.io/ucsc_cgl/samtools " \
```

Comment:
This is a thing you can do, and it's functionally correct, but can we open a ticket to replace this code with something that doesn't involve us munging samtools output? It just seems very fragile.